app/assets/javascripts/discourse/app/components/sidebar/hamburger-dropdown.gjs

```diff
@@ -1,8 +1,10 @@
 import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
```

Comment:
```suggestion
```

Reply:
<img width="906" alt="Screenshot 2024-05-29 at 10 17 35 AM" src="https://github.com/discourse/discourse/assets/1555215/7972917b-1d46-4910-ba1d-09169abdc641">


 😿 my auto linter is so lame..